server/belga/search_providers.py

```diff
@@ -274,7 +277,7 @@ def find(self, query, params=None):
 
             api_params['assetType'] = ' OR '.join([
                 _type for _type in self.TYPE_SUPPORT
```

Comment:
I think it might be good to have some defaults without related items

Reply:
Do you mean if in params we do not have any `type` then we just have to set some default? Currently, defaults are all content types if `type` is not in params -
>    **'Text', 'Brief', 'Alert', 'Short', 'Coverage', 'RelatedDocument', 'RelatedArticle', 'Audio', 'Picture', 'SMS', 'Video'**

should we set default only below four types?

> **'Text'**, **'Brief'**, **'Alert'**, **'Short'**

Reply:
yep, but we should probably rather make those selected by default in the UI so it's more obvious

Reply:
means all the content types(Text, Brief, RelatedArticle etc) should be selected by default in UI when we open Belga Archive search provider right @petrjasek?

Reply:
right